index.css

```diff
@@ -0,0 +1,225 @@
+html,body {
+    margin: 10px;
+    background-color: #ffeead;
+  }
+  
+      .wrapper {
+          display: grid;
+          grid-gap: 5px;
+          grid-template-columns: repeat(12, [col] 1fr) ;
+          /* grid-template-rows: repeat(6, [row] auto ); */
+          grid-template-rows: auto 150px 150px 150px 150px auto;
+          background-color: #ffeead;
+          color: #444;
+      }
+  
+      .box {
+          padding: 10px;
+          font-size: 150%;
+      }
+  
+      .box .box {
+          font-size: 1em;
```

Comment:
what is the sense of setting 1em font-size here? You already have font-size 150% for all .box elements. 
1em of 150% is still 150%. Remove it